core/src/main/java/com/sequenceiq/distrox/v1/distrox/controller/DistroXUpgradeV1Controller.java

```diff
@@ -114,25 +114,29 @@ public DistroXUpgradeV1Response prepareClusterUpgradeByCrn(@ValidCrn(resource =
     @Override
     @InternalOnly
     public DistroXUpgradeV1Response upgradeClusterByNameInternal(@ResourceName String clusterName, @Valid DistroXUpgradeV1Request distroxUpgradeRequest,
-            @InitiatorUserCrn String initiatorUserCrn) {
+            @InitiatorUserCrn String initiatorUserCrn, Boolean rollingUpgradeEnabled) {
```

Comment:
Please incorporate this flag into the `InternalUpgradeSettings` and also alter the `UpgradeConverter`